cmake/scripts/generate_config.cmake

```diff
@@ -54,4 +55,14 @@ file(MAKE_DIRECTORY ${config_install_dir})
 configure_file(${CMAKE_CURRENT_LIST_DIR}/../templates/oneDALConfig.cmake.in ${config_install_dir}/oneDALConfig.cmake @ONLY)
 configure_file(${CMAKE_CURRENT_LIST_DIR}/../templates/oneDALConfigVersion.cmake.in ${config_install_dir}/oneDALConfigVersion.cmake @ONLY)
 
+set(FILE_PATH ${config_install_dir}/oneDALConfig.cmake)
+# Read the content of the input file
+file(READ "${FILE_PATH}" INPUT_CONTENT)
+
+# Perform string substitution
+string(REPLACE "lib/intel64" "lib/${SUB_DIR}" MODIFIED_CONTENT "${INPUT_CONTENT}")
```

Comment:
It feels like the string substitution should not be necessary, and the right subdirectory should be written into the string, from the oneDALConfig.cmake already

Reply:
yes, it seems it's better to setup oneDALConfig properly and not do updates on the fly